src/main/java/com/sweetwith/dailynote/domain/file/File.java

```diff
@@ -0,0 +1,26 @@
+package com.sweetwith.dailynote.domain.file;
+
+import com.sweetwith.dailynote.domain.BaseTimeEntity;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Builder
+@Getter
+public class File extends BaseTimeEntity
+{
+    @Id
+    private Long fileId;
+
+    @Column(nullable = false) private String fileName;
```

Comment:
Are there any rules for determining file names and paths??

Reply:
No. File name and paths are input anywhere writer wants.